Packages/MIES/MIES_DAEphys.ipf

```diff
@@ -1683,18 +1683,36 @@ Function DAP_SweepRollback(string paneltitle, variable sweepNo, variable newSwee
 	ASSERT(rollbackCountNum == rollbackCountText, "Invalid rollback count")
 End
 
-static Function DAP_UpdateSweepLimitsAndDisplay(panelTitle)
-	string panelTitle
+static Function DAP_UpdateSweepLimitsAndDisplay(string panelTitle, [variable initial])
 
 	string panelList
 	variable sweep, nextSweep, maxNextSweep, numPanels, i
 
 	panelList = GetListofLeaderAndPossFollower(panelTitle)
 
-	if(DAP_DeviceIsLeader(panelTitle))
-		sweep = DAG_GetNumericalValue(panelTitle, "SetVar_Sweep")
+	if(ParamIsDefault(initial))
```

Comment:
initial = ParamIsDefault(initial) ? 0 : !!initial

would be shorter.

Reply:
Yeep, but I tend to think that if/else is more readable.